apps/src/code-studio/components/progress/teacherPanel/TeacherPanel.jsx

```diff
@@ -183,13 +187,14 @@ class TeacherPanel extends React.Component {
           )}
           {displayLevelExamples && (
             <div style={styles.exampleSolutions}>
-              {exampleSolutions.map((example, index) => (
+              {exampleSolutions.map((exampleSolutionURL, index) => (
                 <Button
-                  __useDeprecatedTag
                   key={index}
                   text={i18n.exampleSolution({number: index + 1})}
                   color="blue"
-                  href={example}
+                  onClick={() => {
```

Comment:
Hey Alice,
Since this button open a new url when clicked - it should remain <a> tag.
It's not very clear from ticket description, but we should only get rid of `__useDeprecatedTag` when it renders a clickable div (it's easy to understand - it should be button with `__useDeprecatedTag` and `onClick`) so I'm not sure we actually need the ticket for this one specific button.
For this button I think we should keep everything as it was before cause we can only render an `<a>` tag if pass `__useDeprecatedTag` to Button component.

In the future we either add support of `a` tag to current Button component without `__useDeprecatedTag` or create a new one

Reply:
Hi Denys! Thanks for your feedback. I created the ticket for example solutions button being aware that it didn't have an `onClick` property.  so that I added one. My reasoning is to be consistent with the 'Unsubmit' button which is located right above it. I apologize for the confusion - I should've explained that in the PR description - I just updated the [jira ticket](https://codedotorg.atlassian.net/browse/A11Y-236).

I see that a button clicked opening a new URL is also done [here](https://github.com/code-dot-org/code-dot-org/blob/168cd54c20a094a4780e9ca4b88eb59722fa2a85/apps/src/applab/ExternalRedirectDialog.jsx#L59) in `ExternalRedirectButton.jsx` and [here](https://github.com/code-dot-org/code-dot-org/blob/168cd54c20a094a4780e9ca4b88eb59722fa2a85/apps/src/code-studio/pd/application_dashboard/cohort_view.jsx#L118) in `cohort_view.jsx`. I'm going to check in with @code-dot-org/teacher-tools  to see what they think. Thank you!

Reply:
I believe if a button opens a new page, it should use an `a` tag in order to be accessible. I'd rather figure out how to restyle one of the buttons to match the other -- @hannahbergam has some experience here

Reply:
Thanks Denys and Bethany for the clarification - am only removing tag from 'Unsubmit' button.